docs/manager/graphql-reference/schema.graphql

```diff
@@ -770,6 +779,14 @@
   cursor: String!
 }
 
+"""Added in 25.2.0."""
+scalar ContainerRegistryScopeField
+
+"""
+Added in 25.2.0. One of ['read_attribute', 'read_sensitive_attribute', 'update_attribute', 'delete_project', 'associate_with_user'].
+"""
+scalar GroupPermissionField
```

Comment:
It's better to use GraphQL's enum type for this kind of value. https://graphql.org/learn/schema/#enum-types

Reply:
This will be resolved in separate follow-up PR